src/Sylius/Bundle/ApiBundle/Assigner/OrderPromoCodeAssigner.php

```diff
@@ -11,44 +11,31 @@
 
 declare(strict_types=1);
 
-namespace Sylius\Bundle\ApiBundle\CommandHandler\Cart;
+namespace Sylius\Bundle\ApiBundle\Assigner;
 
-use Sylius\Bundle\ApiBundle\Command\Cart\ApplyCouponToCart;
 use Sylius\Component\Core\Model\OrderInterface;
-use Sylius\Component\Core\Repository\OrderRepositoryInterface;
 use Sylius\Component\Order\Processor\OrderProcessorInterface;
 use Sylius\Component\Promotion\Model\PromotionCouponInterface;
 use Sylius\Component\Promotion\Repository\PromotionCouponRepositoryInterface;
-use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
 use Webmozart\Assert\Assert;
 
-/** @experimental */
-final class ApplyCouponToCartHandler implements MessageHandlerInterface
```

Comment:
Handler and command removal should be noted in UPGRADE-API-1.11.md. Something like `ApplyCouponToCart` has been replaced with `UpdateOrder` command and `ApplyCouponToCartHandler` was replaced with `UpdateOrderHandler`